src/bin/sse-test-api/main.rs

```diff
@@ -187,14 +187,15 @@ async fn main() -> std::io::Result<()> {
     .bind("127.0.0.1:8080")?
     .run();
 
+    let handle = server.handle();
```

Comment:
actix-web isn't pinned (we don't commit the lock) so I had to change how this works.